docs/docs/references/enhanced-security.md

```diff
@@ -0,0 +1,69 @@
+## Nebari Security Considerations
+
+The security of _AWS Nebari_ deployments can be enhanced through the following deployment configuration options in `nebari-config.yaml`:
+
+- **Explicit definition of container sources**  
+  This option allows for the use of locally mirrored, security-hardened, or otherwise customized container images in place of the containers used by default.
+  See: [container-sources](container-sources.md)
+
+- **Definition of an ssh key that can access EKS hosts**  
+  EKS hosts by default cannot be accessed via ssh. This configuration item allows ssh access into EKS hosts, which can be useful for troubleshooting or external monitoring and auditing purposes.
+
+```
+amazon_web_services:
```

Comment:
@tylergraff 
The following feature, although [tested](https://github.com/nebari-dev/nebari/compare/main...MetroStar:nebari:eks-private-cluster), was never merged
Lines 9-15
```
amazon_web_services:
  ec2_keypair_name: [example_keypair_name] # Name, not ARN
```

Reply:
fixed!